frontend/public/src/components/EnrolledItemCard.js

```diff
@@ -356,14 +339,10 @@ export class EnrolledItemCard extends React.Component<
                 </DropdownMenu>
               </Dropdown>
             </div>
-
-            <div className="d-flex justify-content-between align-content-start flex-nowrap mb-3">
-              <h2 className="my-0 mr-3">{title}</h2>
-            </div>
             <div className="detail">
-              {courseId} |{" "}
+              {courseId}
               {startDateDescription !== null && startDateDescription.active ? (
-                <span>Starts - {startDateDescription.datestr}</span>
+                <span>|{" "}Starts - {startDateDescription.datestr}</span>
```

Comment:
This results in there not being a space between the readable ID and the start date (when there is one).